Tests/Unit/ExtensionTest.php

```diff
@@ -79,14 +78,21 @@ private function getValidConfiguration(array $options): array
                 'resource_server' => [
                     'public_key' => 'foo',
                 ],
-                'persistence' => [],
+                //Pick one for valid config:
+                //'persistence' => ['doctrine' => []]
+                'persistence' => ['in_memory' => 1],
```

Comment:
This change seems unrelated to Symfony 5.

Reply:
this change was helpful to me though, when trying to understand how to configure the package in my own project, it helped me validate what an in-memory persistence should be configured like. This is an actual proper configuration of the extension, because the test had an artificial manual call to "$container->setAlias(ScopeManagerInterface::class, ScopeManager::class);" which was only making the test pass artificially without signaling what a proper configuration should look like.

I would recommend keeping this.

Reply:
I could revert this and the one below, if you still wish me to, and put it in a later PR. Let me know @X-Coder264

Reply:
@elchris This is fine actually. :+1: